New intro to d mri module

Davi1990 opened the pull request:
This is the PR for the intro to dMRI module. I reviewed the materials from the Neuroimaging Carpentry series and make it run locally. 

Looking forward to hear your feedback on that 

Cheers

Dave

htwangtw commented on content/en/modules/introduction_to_dMRI/index.md:
```diff
@@ -0,0 +1,179 @@
+
+The prerequisites to take this module are:
+ * Installations
+ * [Introduction to dMRI](https://psy6983.brainhackmtl.org/modules/introduction_to_dMRI/) module.
```
This URL will fail once merged. Can we use relative path? I would need someone with more understanding of the Hugo framework to check.(@complexbrains ?)

clarkenj replied:
This is fixed.

htwangtw reviewed (changes requested):
Thank you for submitting this PR :tada: 
Two main things for house keeping.
- Can you remove the mne python part to aseparate PR so it's easier to review?
- Have you build the website locally with the [instruction](https://github.com/school-brainhack/school-brainhack.github.io#run-locally)? This will really save time for reviews.

Davi1990 commented:
> Thank you for submitting this PR 🎉 Two main things for house keeping.
> 
> * Can you remove the mne python part to aseparate PR so it's easier to review?
> * Have you build the website locally with the [instruction](https://github.com/school-brainhack/school-brainhack.github.io#run-locally)? This will really save time for reviews.

htwangtw commented:
> that's exactly what I did but for some reason the first was included in this as well.

You need to delete the redundant files on this branch, commit and push the change:

```
git checkout new_intro_to_dMRI_module
rm -rf content/en/modules/mne_python/
git add content/en/modules/
git commit -m "remove redundant mne python content"
git push
```

> I followed the instruction and hugo is running but I am not sure what you are asking

Please check if the page you added can compile correctly with no formatting errors.
Remember to compile the branch where the changes are in. This is how you switch to an existing branch:
```
git checkout new_intro_to_dMRI_module
```

Davi1990 commented:
Hey @htwangtw, 

- I removed the `mne python` part so it should be easier to review
- I added in the instruction the part for building the virtual environment as requested
- I was able to run the website locally using `hugo serve -D`. No error where displayed! Only concern I have is that if I tried to go to the `http://localhost:1313/modules/Intro_to_dMRI_workshop/` it is actually showing this: 

![image](https://user-images.githubusercontent.com/37826571/233128573-a7e69b25-4d1a-4ae4-b5a8-30eda315bb42.png)

Whereas if I tried to go to other modules (e.g. datalad): 

![image](https://user-images.githubusercontent.com/37826571/233128774-237967e8-2762-4cb2-880d-ad426b051c41.png)

It is weird because the `index.md` looks ok to me. 

Please let me know your thought regarding this

htwangtw commented:
> * No error where displayed! Only concern I have is that if I tried to go to the `http://localhost:1313/modules/Intro_to_dMRI_workshop/` it is actually showing this:

This is a funny statement, as I clearly see this as a bug / unwanted behaviour / formatting error on the pages you work on :sweat_smile: 

If the datalad page is working you can have a look at how things are coded / referenced there and change things on your side.

Also some house keeping - can you make sure the directory name (`Intro_to_dMRI_workshop`) is all lower case. Perhaps `dmri_intro`? Just to be consistent with the style of the rest of the directories.

Davi1990 commented:
I am happy to make to lough then. Jokes apart what I meant is that it is building the page without error

htwangtw commented:
Can you commit only the content instead of adding all changes to the commit?
We don't want the hugo lock file or the submodule update....

clarkenj reviewed (changes requested):
Hi @Davi1990, this is fantastic, thank you! I left some comments, do you think you could address them by early next week, then we can merge?

Davi1990 commented:
Hey @clarkenj. I apologize I made the changes you requested but I am encountering multiple conflicts. Since these are small changes, is there any chance you can apply them? I can share the new index.md file in case

clarkenj commented:
> Hey @clarkenj. I apologize I made the changes you requested but I am encountering multiple conflicts. Since these are small changes, is there any chance you can apply them? I can share the new index.md file in case

Hi @Davi1990, thank you so much for your quick response! Yes I can make the changes in that case if you agreed with all my comments?

clarkenj commented:
@htwangtw I am having trouble merging this because there is a conflict with `content/en/modules/mne_python/index.md`, but Davide had already removed that file from this PR... are you able to help?

htwangtw commented:
I will have a look and I still believe there are a lot of formatting issue to be resolved (lots of syntax violating markrdown). I will have a look and resolve those

clarkenj commented:
Ah OK, sorry I missed those. Thank you!

clarkenj reviewed (commented):
I am having issues installing the required packages and I can't seem to solve them. Did the requirements.txt work as is for you Hao-Ting?

clarkenj commented:
> > I am having issues installing the required packages and I can't seem to solve them. Did the requirements.txt work as is for you @htwangtw?
> 
> yes the requirement file is fine when i install it in a fresh environment. ~However I do have problem running the single subject fetching....~ Just really slow

Odd, I'm also in a fresh environment. Everything else looks good, should we merge? Or perhaps ask Isil to review as well?

htwangtw commented:
I think this is good to merge. Can you approve the pr through the review system? @clarkenj

clarkenj reviewed (approved):
LGTM!
